cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,149 @@
 /// <reference types='cypress' />
+const faker = require('faker');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  const worker = {
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    email: faker.internet.email(),
+    age: Math.ceil(Math.random() * 30),
+    salary: Math.ceil(Math.random() * 1000),
+    department: faker.name.lastName()
+  };
+  const worker1 = {
+    firstName: 'Cierra',
+    lastName: 'Vega',
+    email: 'cierra@example.com',
+    age: 39,
+    salary: 10000,
+    department: 'Insurance',
+    newFirstName: faker.name.firstName()
+  };
```

Comment:
I think it will be better if you do like the previous worker with dynamic variables